examples/graph-db-demo/neptune-example.ipynb

```diff
@@ -42,7 +42,8 @@
     "    \"embedder\": {\n",
     "        \"provider\": \"aws_bedrock\",\n",
     "        \"config\": {\n",
-    "            \"model\": \"amazon.titan-embed-text-v2:0\"\n",
+    "            \"model\": \"amazon.titan-embed-text-v2:0\",\n",
+    "            \"embedding_dims\": 1024\n",
```

Comment:
We should have a note that the vector dimensions defined in the Neptune Analytics store must match the embedding dimensions.